renku/cli/rerun.py

```diff
@@ -28,14 +28,17 @@
 
 .. code-block:: text
 
-    A-(step 1)-B-(step 2*)-C
+    A-[step 1]-B-[step 2*]-C
 
 Recreate a specific output file by running:
 
   .. code-block:: console
 
      $ renku rerun C
 
+If you would like to recreate a file which was produced by a tool together
```

Comment:
Clearer wording:

    If you would like to recreate a file which was one of several produced by a tool, 
    then these files must be recreated as well. See the explanation in 
    :ref:`updating siblings <cli-update-with-siblings>`.